parsers/forth.c

```diff
@@ -0,0 +1,78 @@
+/*
+*   Copyright (c) 2023, Eric Forgeot
+*
+*   Based on work by Jon Strait
```

Comment:
Any references (e.g. URL) for the original work?

Reply:
I've put it in the newest commit (it was for markdown ctags in geany, it seems to have been replaced later but I've found the original code again)